packages/server/graphql/mutations/setMeetingSettings.ts

```diff
@@ -58,17 +60,20 @@ const setMeetingSettings = {
             checkinEnabled ? row('phaseTypes') : row('phaseTypes').difference(['checkin']),
             checkinEnabled ? row('phaseTypes').prepend('checkin') : row('phaseTypes')
           )
+          Object.assign(meetingSettings, {hasIcebreaker: checkinEnabled})
```

Comment:
-1 If you type it properly, you can just do `meetingSettings.hasIcebreaker = checkinEnabled` here which is more readable.